pkg/api/vm/formatter.go

```diff
@@ -16,21 +16,22 @@ import (
 )
 
 const (
-	startVM        = "start"
-	stopVM         = "stop"
-	restartVM      = "restart"
-	softReboot     = "softreboot"
-	pauseVM        = "pause"
-	unpauseVM      = "unpause"
-	ejectCdRom     = "ejectCdRom"
-	migrate        = "migrate"
-	abortMigration = "abortMigration"
-	backupVM       = "backup"
-	restoreVM      = "restore"
-	createTemplate = "createTemplate"
-	addVolume      = "addVolume"
-	removeVolume   = "removeVolume"
-	cloneVM        = "clone"
+	startVM            = "start"
+	stopVM             = "stop"
+	restartVM          = "restart"
+	softReboot         = "softreboot"
+	pauseVM            = "pause"
+	unpauseVM          = "unpause"
+	ejectCdRom         = "ejectCdRom"
+	migrate            = "migrate"
+	abortMigration     = "abortMigration"
+	getMigratableNodes = "getMigratableNodes"
```

Comment:
the http method of actions is `POST`, but the name `getMigratableNodes` seems like a GET API.
maybe `findMigratableNodes` would be better？